Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -0,0 +1,26 @@
+## How To Submit Content to EOS Documentation Portal
+
+**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
+
+Complete the following steps to submit your content to DevRel:
+
+1. Create a PR in the DevRel repository
+   Include the following information in the ticket/issue (if applicable)
+   * Your Name
+   * Title of document
+   * Document type (template applied, if applicable)
+     * Overview
+     * Explainer
+     * Tutorial
+     * How To
+   * Who (what role) the document pertains to
+     * App Developer
+     * Node Operator
+     * Smart Contract Developer
+   * Considerations
+     * Considerations are special instructions to DevRel, such as an indication the content needs more information, request to be uploaded to the EOS Documentation portal by a specific date, …
```

Comment:
These bullet items are for github issues, not PRs, which are not applicable here to these instructions. Please remove.

Reply:
Done